envi/bits.py

```diff
@@ -235,8 +235,8 @@ def buildbytes(value, size, bigend=False):
 def byteswap(value, size):
     ret = 0
     for i in range(size):
-        ret |= (value >> (8*i)) & 0xff
         ret = ret << 8
+        ret |= (value >> (8*i)) & 0xff
```

Comment:
So we have a envi/tests/test_bits.py file where it would be good to add tests to show that this is correct :smile: